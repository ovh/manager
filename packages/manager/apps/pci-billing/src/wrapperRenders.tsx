import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
import {
  ShellContext,
  ShellContextType,
} from '@ovh-ux/manager-react-shell-client';
import { vi } from 'vitest';

export const shellContext = {
  environment: {
    getUser: () => ({ ovhSubsidiary: 'FR', currency: 'â‚¬' }),
  },
  shell: {
    navigation: {
      getURL: vi.fn(),
    },
  },
};

const queryClient = new QueryClient({
  defaultOptions: {
    mutations: {
      retry: false,
    },
    queries: {
      retry: false,
    },
  },
});

export const wrapper = ({ children }) => (
  <QueryClientProvider client={queryClient}>
    <ShellContext.Provider
      value={(shellContext as unknown) as ShellContextType}
    >
      {children}
    </ShellContext.Provider>
  </QueryClientProvider>
);
