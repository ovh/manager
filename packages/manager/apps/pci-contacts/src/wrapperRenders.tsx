import {
  ShellContext,
  ShellContextType,
} from '@ovh-ux/manager-react-shell-client';
import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
import { ReactNode } from 'react';
import { MemoryRouter } from 'react-router-dom';
import { vi } from 'vitest';

export const shellContext = ({
  environment: {
    getUser: () => ({ ovhSubsidiary: 'FR' }),
    getRegion: () => 'EU',
  },
  shell: {
    navigation: {
      navigateTo: vi.fn(),
      getURL: vi.fn().mockImplementation((app, path) => {
        if (app === 'dedicated' && path === '#/billing/history') {
          return Promise.resolve('https://billing-url');
        }
        return Promise.resolve('');
      }),
    },
  },
} as unknown) as ShellContextType;

export const createWrapper = (
  contextValue: ShellContextType = shellContext,
) => {
  const queryClient = new QueryClient({
    defaultOptions: {
      queries: {
        staleTime: 300_000,
      },
    },
  });

  const Wrapper = ({ children }: { children: ReactNode }) => (
    <QueryClientProvider client={queryClient}>
      <ShellContext.Provider value={contextValue}>
        <MemoryRouter>{children}</MemoryRouter>
      </ShellContext.Provider>
    </QueryClientProvider>
  );
  return Wrapper;
};
