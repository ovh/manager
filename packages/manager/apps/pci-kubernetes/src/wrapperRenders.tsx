import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
import {
  ShellContext,
  ShellContextType,
} from '@ovh-ux/manager-react-shell-client';
import { vi } from 'vitest';

export const shellContext = {
  environment: {
    getRegion: vi.fn().mockReturnValue('EU'),
    getUser: () => ({
      ovhSubsidiary: 'mocked_ovhSubsidiary',
      currency: { code: 'EUR' },
    }),
  },
  shell: {
    navigation: {
      getURL: vi.fn(),
    },
  },
};

const queryClient = new QueryClient({
  defaultOptions: {
    queries: {
      retry: false,
    },
  },
});

export const wrapper = ({ children }) => (
  <QueryClientProvider client={queryClient}>
    <ShellContext.Provider
      value={(shellContext as unknown) as ShellContextType}
    >
      {children}
    </ShellContext.Provider>
  </QueryClientProvider>
);
