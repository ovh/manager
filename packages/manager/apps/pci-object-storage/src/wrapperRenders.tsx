import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
import {
  ShellContext,
  ShellContextType,
} from '@ovh-ux/manager-react-shell-client';
import { vi } from 'vitest';
import { createMemoryRouter, RouterProvider } from 'react-router-dom';
import React from 'react';

export const shellContext = {
  environment: {
    getUser: () => ({ ovhSubsidiary: 'mocked_ovhSubsidiary' }),
  },
  shell: {
    navigation: {
      getURL: vi.fn(),
    },
    tracking: {
      trackPage: vi.fn(),
      trackClick: vi.fn(),
    },
  },
};

const queryClient = new QueryClient({
  defaultOptions: {
    mutations: {
      retry: false,
    },
    queries: {
      retry: false,
    },
  },
});

export const wrapper = ({ children }) => (
  <QueryClientProvider client={queryClient}>
    <ShellContext.Provider
      value={(shellContext as unknown) as ShellContextType}
    >
      {children}
    </ShellContext.Provider>
  </QueryClientProvider>
);

export const mockShellContext = {
  environment: {
    getRegion: vi.fn().mockReturnValue('EU'),
    getUser: vi.fn().mockReturnValue({ ovhSubsidiary: 'mocked_ovhSubsidiary' }),
    getApplication: vi.fn(),
  },
  shell: {
    environment: {
      getRegion: vi.fn().mockReturnValue('EU'),
      getUser: vi.fn().mockReturnValue({}),
      getApplication: vi.fn(),
    },
    navigation: {
      getURL: vi.fn(),
    },
    tracking: {
      trackPage: vi.fn(),
      trackClick: vi.fn(),
    },
  },
};

export const wrapperShow = ({ children }: { children: React.ReactNode }) => {
  const router = createMemoryRouter(
    [
      {
        path: '/',
        element: children,
        handle: {
          crumb: () => 'Home',
        },
      },
    ],
    {
      initialEntries: ['/'],
      future: {
        v7_startTransition: true,
        v7_relativeSplatPath: true,
      } as any,
    },
  );

  return (
    <QueryClientProvider client={queryClient}>
      <ShellContext.Provider value={mockShellContext as any}>
        <RouterProvider router={router} />
      </ShellContext.Provider>
    </QueryClientProvider>
  );
};

export const wrapperOffsiteReplication = ({
  children,
}: {
  children: React.ReactNode;
}) => (
  <QueryClientProvider client={queryClient}>
    <ShellContext.Provider value={mockShellContext as any}>
      {children}
    </ShellContext.Provider>
  </QueryClientProvider>
);
