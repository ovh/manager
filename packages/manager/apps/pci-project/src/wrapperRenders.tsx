import {
  ShellContext,
  ShellContextType,
} from '@ovh-ux/manager-react-shell-client';
import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
import { ReactNode } from 'react';
import { MemoryRouter } from 'react-router-dom';
import { vi } from 'vitest';

export const shellContext = ({
  environment: {
    getUser: () => ({ ovhSubsidiary: 'FR' }),
    getRegion: () => 'EU',
  },
  shell: {
    navigation: {
      navigateTo: vi.fn(),
      getURL: vi.fn().mockImplementation((app, path) => {
        if (app === 'dedicated' && path === '#/billing/history') {
          return Promise.resolve('https://billing-url');
        }
        if (app === 'dedicated' && path === '#/contacts/services') {
          return Promise.resolve('https://contacts-url');
        }
        return Promise.resolve('https://mocked-url');
      }),
    },
  },
} as unknown) as ShellContextType;

export const createWrapper = (
  contextValue: ShellContextType = shellContext,
) => {
  const queryClient = new QueryClient({
    defaultOptions: {
      queries: {
        staleTime: 300_000,
      },
    },
  });

  const Wrapper = ({ children }: { children: ReactNode }) => (
    <QueryClientProvider client={queryClient}>
      <ShellContext.Provider value={contextValue}>
        <MemoryRouter>{children}</MemoryRouter>
      </ShellContext.Provider>
    </QueryClientProvider>
  );
  return Wrapper;
};
