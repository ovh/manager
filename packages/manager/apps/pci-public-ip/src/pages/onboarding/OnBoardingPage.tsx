import {
  useEnvironment,
  useNavigation,
} from '@ovh-ux/manager-react-shell-client';
import {
  Card,
  OnboardingLayout,
  PciDiscoveryBanner,
  isDiscoveryProject,
} from '@ovhcloud/manager-components';
import { OsdsBreadcrumb, OsdsText } from '@ovhcloud/ods-components/react';
import { useEffect, useState } from 'react';
import { useTranslation } from 'react-i18next';
import {
  Outlet,
  useNavigate,
  useParams,
  useRouteLoaderData,
} from 'react-router-dom';
import {
  ODS_THEME_COLOR_INTENT,
  ODS_THEME_TYPOGRAPHY_SIZE,
} from '@ovhcloud/ods-common-theming';
import {
  ODS_TEXT_LEVEL,
  OdsBreadcrumbAttributeItem,
} from '@ovhcloud/ods-components';
import { Project } from '@/api/data/project';
import { useAllFloatingIP } from '@/api/hooks/useFloatingIP';
import { GUIDES } from './onboarding.constants';

export default function OnBoardingPage() {
  const { t } = useTranslation();
  const { t: tOnBoarding } = useTranslation('onboarding');
  const { projectId } = useParams();
  const navigation = useNavigation();
  const { ovhSubsidiary } = useEnvironment().getUser();
  const project = useRouteLoaderData('public-ips') as Project;
  const [urlProject, setUrlProject] = useState('');
  const navigate = useNavigate();
  const { data: floatingIPs, isLoading } = useAllFloatingIP(projectId);

  useEffect(() => {
    navigation
      .getURL('public-cloud', `#/pci/projects/${projectId}`, {})
      .then((data) => {
        setUrlProject(data as string);
      });
  }, [projectId, navigation]);

  useEffect(() => {
    // if there are users redirect to user listing page
    if (!isLoading && floatingIPs.length > 0) {
      navigate(`/pci/projects/${projectId}/public-ips`);
    }
  }, [isLoading, floatingIPs, navigate]);

  const breadcrumbItems: OdsBreadcrumbAttributeItem[] = [
    {
      href: urlProject,
      label: project?.description,
    },
    {
      label: t('pci_additional_ips_title'),
    },
  ];

  const tileItems = [
    {
      id: 'create-delete-openstack-user',
      texts: {
        title: tOnBoarding(
          'pci_additional_ips_onboarding_guides_additional_public_ips_title',
        ),
        description: tOnBoarding(
          'pci_additional_ips_onboarding_guides_additional_public_ips_description',
        ),
        category: tOnBoarding('onboarding_guide_title'),
      },
      href: GUIDES.additional_public_ips[ovhSubsidiary],
    },
    {
      id: 'configure-horizon-user-access',
      texts: {
        title: tOnBoarding(
          'pci_additional_ips_onboarding_guides_configure_additional_ip_title',
        ),
        description: tOnBoarding(
          'pci_additional_ips_onboarding_guides_configure_additional_ip_description',
        ),
        category: tOnBoarding('onboarding_guide_title'),
      },
      href: GUIDES.configure_additional_ip[ovhSubsidiary],
    },
    {
      id: 'prepare-openstack-api-env',
      texts: {
        title: tOnBoarding(
          'pci_additional_ips_onboarding_guides_configure_floating_ip_title',
        ),
        description: tOnBoarding(
          'pci_additional_ips_onboarding_guides_configure_floating_ip_description',
        ),
        category: tOnBoarding('onboarding_guide_title'),
      },
      href: GUIDES.configure_floating_ip[ovhSubsidiary],
    },
  ];

  return (
    <>
      {project && <OsdsBreadcrumb items={breadcrumbItems} />}
      {isDiscoveryProject(project) && (
        <div className="mb-8">
          <PciDiscoveryBanner projectId={projectId} />
        </div>
      )}
      <OnboardingLayout
        title={t('pci_additional_ips_title')}
        description={
          <>
            <OsdsText
              color={ODS_THEME_COLOR_INTENT.text}
              level={ODS_TEXT_LEVEL.body}
              size={ODS_THEME_TYPOGRAPHY_SIZE._500}
            >
              {tOnBoarding('pci_additional_ips_onboarding_content1')}
            </OsdsText>
            <OsdsText
              color={ODS_THEME_COLOR_INTENT.text}
              level={ODS_TEXT_LEVEL.body}
              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
              className="mt-4 block"
            >
              {tOnBoarding('pci_additional_ips_onboarding_content2')}
            </OsdsText>
            <OsdsText
              color={ODS_THEME_COLOR_INTENT.text}
              level={ODS_TEXT_LEVEL.body}
              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
              className="mt-6 block"
            >
              {tOnBoarding('pci_additional_ips_onboarding_content3')}
            </OsdsText>
            <OsdsText
              color={ODS_THEME_COLOR_INTENT.text}
              level={ODS_TEXT_LEVEL.body}
              size={ODS_THEME_TYPOGRAPHY_SIZE._500}
              className="block mt-6"
            >
              {tOnBoarding('pci_additional_ips_onboarding_content4_heading')}
            </OsdsText>
            <OsdsText
              color={ODS_THEME_COLOR_INTENT.text}
              level={ODS_TEXT_LEVEL.body}
              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
              className="block"
            >
              {tOnBoarding(
                'pci_additional_ips_onboarding_content4_description',
              )}
            </OsdsText>
            <OsdsText
              color={ODS_THEME_COLOR_INTENT.text}
              level={ODS_TEXT_LEVEL.body}
              size={ODS_THEME_TYPOGRAPHY_SIZE._500}
              className="block mt-6"
            >
              {tOnBoarding('pci_additional_ips_onboarding_content5_heading')}
            </OsdsText>
            <OsdsText
              color={ODS_THEME_COLOR_INTENT.text}
              level={ODS_TEXT_LEVEL.body}
              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
              className="block"
            >
              {tOnBoarding(
                'pci_additional_ips_onboarding_content5_description',
              )}
            </OsdsText>
          </>
        }
        orderButtonLabel={tOnBoarding(
          'pci_additional_ips_onboarding_action_buy',
        )}
        onOrderButtonClick={() => {
          if (!isDiscoveryProject(project)) navigate('./new');
        }}
        isActionDisabled={isDiscoveryProject(project)}
      >
        <aside className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-8 pt-12">
          {tileItems.map((tile) => (
            <Card key={tile.id} href={tile.href} texts={tile.texts} />
          ))}
        </aside>
      </OnboardingLayout>
      <Outlet />
    </>
  );
}
