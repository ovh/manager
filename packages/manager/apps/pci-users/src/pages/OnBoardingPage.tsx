import { useNavigation } from '@ovh-ux/manager-react-shell-client';
import { Card, OnboardingLayout } from '@ovhcloud/manager-components';
import { OdsBreadcrumbAttributeItem } from '@ovhcloud/ods-components';
import { OsdsBreadcrumb } from '@ovhcloud/ods-components/react';
import { useEffect, useState } from 'react';
import { useTranslation } from 'react-i18next';
import {
  Outlet,
  useHref,
  useParams,
  useRouteLoaderData,
} from 'react-router-dom';
import { Project } from '@/data/project';

export default function OnBoardingPage() {
  const { t } = useTranslation('common');
  const { projectId } = useParams();
  const navigation = useNavigation();
  const project = useRouteLoaderData('users') as Project;
  const [urlProject, setUrlProject] = useState('');
  const hrefAddUser = useHref('./new');

  useEffect(() => {
    navigation
      .getURL('public-cloud', `#/pci/projects/${projectId}`, {})
      .then((data) => {
        setUrlProject(data as string);
      });
  }, [projectId, navigation]);

  const breadcrumbItems: OdsBreadcrumbAttributeItem[] = [
    {
      href: urlProject,
      label: project?.description,
    },
    {
      label: t('pci_projects_project_users_title'),
    },
  ];

  const tileItems = [
    {
      id: 'create-delete-openstack-user',
      texts: {
        title: t(
          'pci_projects_project_users_onboarding_guides_create-delete-openstack-user_title',
        ),
        description: t(
          'pci_projects_project_users_onboarding_guides_create-delete-openstack-user_description',
        ),
        category: t('onboarding_guide_title'),
      },
      href:
        'https://docs.ovh.com/gb/en/public-cloud/creation-and-deletion-of-openstack-user/',
    },
    {
      id: 'configure-horizon-user-access',
      texts: {
        title: t(
          'pci_projects_project_users_onboarding_guides_configure-horizon-user-access_title',
        ),
        description: t(
          'pci_projects_project_users_onboarding_guides_configure-horizon-user-access_description',
        ),
        category: t('onboarding_guide_title'),
      },
      href:
        'https://docs.ovh.com/gb/en/public-cloud/configure_user_access_to_horizon/',
    },
    {
      id: 'prepare-openstack-api-env',
      texts: {
        title: t(
          'pci_projects_project_users_onboarding_guides_prepare-openstack-api-env_title',
        ),
        description: t(
          'pci_projects_project_users_onboarding_guides_prepare-openstack-api-env_description',
        ),
        category: t('onboarding_guide_title'),
      },
      href:
        'https://docs.ovh.com/gb/en/public-cloud/prepare_the_environment_for_using_the_openstack_api/',
    },
  ];

  const descriptionContent = [
    t('pci_projects_project_users_onboarding_content1'),
    t('pci_projects_project_users_onboarding_content2'),
    t('pci_projects_project_users_onboarding_content3'),
    t('pci_projects_project_users_onboarding_content4'),
  ].join(' ');

  return (
    <>
      {project && <OsdsBreadcrumb items={breadcrumbItems} />}
      <OnboardingLayout
        title={t('pci_projects_project_users_onboarding_title')}
        description={descriptionContent}
        /**
         * TODO: Should rename those props to make them generic because the component must be reused in other context different that order
         */
        orderButtonLabel={t(
          'pci_projects_project_users_onboarding_action_label',
        )}
        orderHref={hrefAddUser}
      >
        <aside className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-8 pt-12">
          {tileItems.map((tile) => (
            <Card key={tile.id} href={tile.href} texts={tile.texts} />
          ))}
        </aside>
      </OnboardingLayout>
      <Outlet />
    </>
  );
}
