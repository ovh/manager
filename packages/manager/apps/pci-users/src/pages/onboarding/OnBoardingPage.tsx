import { useNavigation } from '@ovh-ux/manager-react-shell-client';
import { Card, OnboardingLayout } from '@ovh-ux/manager-react-components';
import { OdsBreadcrumbAttributeItem } from '@ovhcloud/ods-components';
import { OsdsBreadcrumb } from '@ovhcloud/ods-components/react';
import { useEffect, useState } from 'react';
import { useTranslation } from 'react-i18next';
import {
  Outlet,
  useNavigate,
  useParams,
  useRouteLoaderData,
} from 'react-router-dom';
import {
  isDiscoveryProject,
  PciDiscoveryBanner,
  TProject,
} from '@ovh-ux/manager-pci-common';
import { useAllUsers } from '@/api/hooks/useUser';

export default function OnBoardingPage() {
  const { t } = useTranslation('common');
  const { projectId } = useParams();
  const navigation = useNavigation();
  const project = useRouteLoaderData('users') as TProject;
  const [urlProject, setUrlProject] = useState('');
  const navigate = useNavigate();
  const { data: users, isLoading } = useAllUsers(projectId);

  useEffect(() => {
    navigation
      .getURL('public-cloud', `#/pci/projects/${projectId}`, {})
      .then((data) => {
        setUrlProject(data as string);
      });
  }, [projectId, navigation]);

  useEffect(() => {
    // if there are users redirect to user listing page
    if (!isLoading && users.length > 0) {
      navigate(`/pci/projects/${projectId}/users`);
    }
  }, [isLoading, users, navigate]);

  const breadcrumbItems: OdsBreadcrumbAttributeItem[] = [
    {
      href: urlProject,
      label: project?.description,
    },
    {
      label: t('pci_projects_project_users_title'),
    },
  ];

  const tileItems = [
    {
      id: 'create-delete-openstack-user',
      texts: {
        title: t(
          'pci_projects_project_users_onboarding_guides_create-delete-openstack-user_title',
        ),
        description: t(
          'pci_projects_project_users_onboarding_guides_create-delete-openstack-user_description',
        ),
        category: t('onboarding_guide_title'),
      },
      href:
        'https://docs.ovh.com/gb/en/public-cloud/creation-and-deletion-of-openstack-user/',
    },
    {
      id: 'configure-horizon-user-access',
      texts: {
        title: t(
          'pci_projects_project_users_onboarding_guides_configure-horizon-user-access_title',
        ),
        description: t(
          'pci_projects_project_users_onboarding_guides_configure-horizon-user-access_description',
        ),
        category: t('onboarding_guide_title'),
      },
      href:
        'https://docs.ovh.com/gb/en/public-cloud/configure_user_access_to_horizon/',
    },
    {
      id: 'prepare-openstack-api-env',
      texts: {
        title: t(
          'pci_projects_project_users_onboarding_guides_prepare-openstack-api-env_title',
        ),
        description: t(
          'pci_projects_project_users_onboarding_guides_prepare-openstack-api-env_description',
        ),
        category: t('onboarding_guide_title'),
      },
      href:
        'https://docs.ovh.com/gb/en/public-cloud/prepare_the_environment_for_using_the_openstack_api/',
    },
  ];

  return (
    <>
      {project && <OsdsBreadcrumb items={breadcrumbItems} />}

      <div className="mb-8">
        <PciDiscoveryBanner project={project} />
      </div>

      <OnboardingLayout
        title={t('pci_projects_project_users_onboarding_title')}
        description={
          <>
            <span>{t('pci_projects_project_users_onboarding_content1')}</span>
            <span className="font-bold block mt-4">
              {t('pci_projects_project_users_onboarding_content2')}
            </span>
            <span>{t('pci_projects_project_users_onboarding_content3')}</span>
            <span className="block mt-4">
              {t('pci_projects_project_users_onboarding_content4')}
            </span>
          </>
        }
        /**
         * TODO: Should rename those props to make them generic because the component must be reused in other context different that order
         */
        orderButtonLabel={t(
          'pci_projects_project_users_onboarding_action_label',
        )}
        onOrderButtonClick={() => {
          if (!isDiscoveryProject(project)) navigate('./new');
        }}
        isActionDisabled={isDiscoveryProject(project)}
      >
        {tileItems.map((tile) => (
          <Card key={tile.id} href={tile.href} texts={tile.texts} />
        ))}
      </OnboardingLayout>
      <Outlet />
    </>
  );
}
