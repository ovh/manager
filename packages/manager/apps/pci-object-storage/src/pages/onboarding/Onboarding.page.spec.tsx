import { render } from '@testing-library/react';
import { useFeatureAvailability } from '@ovh-ux/manager-react-components';
import { vi } from 'vitest';
import OnBoardingPage from './Onboarding.page';
import { wrapper } from '@/wrapperRenders';

describe('OnboardingPage', () => {
  it('sould display correctly', () => {
    vi.mocked(useFeatureAvailability).mockReturnValue({
      data: { feature: false },
      isPending: false,
    } as never);
    const { container } = render(<OnBoardingPage />, { wrapper });
    expect(container).toMatchSnapshot();
  });
});
