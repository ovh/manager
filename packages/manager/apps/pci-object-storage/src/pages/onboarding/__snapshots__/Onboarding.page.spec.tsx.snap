// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`OnboardingPage > sould display correctly 1`] = `
<div>
  <ods-spinner
    data-testid="redirectionGuard_spinner"
    size="md"
  />
</div>
`;
